src/platform/esp32/architecture.h

```diff
@@ -102,8 +102,8 @@
 #define HW_VENDOR meshtastic_HardwareModel_PRIVATE_HW
 #elif defined(NANO_G1)
 #define HW_VENDOR meshtastic_HardwareModel_NANO_G1
-#elif defined(M5STACK)
-#define HW_VENDOR meshtastic_HardwareModel_M5STACK
+#elif defined(M5STACK_CORE)
+#define HW_VENDOR meshtastic_HardwareModel_M5STACK_CORE
 #elif defined(STATION_G1)
```

Comment:
This, on the other hand, needs a change to the protobufs :-) meshtastic_HardwareModel_M5STACK_CORE is not defined there, meshtastic_HardwareModel_M5STACK is. Since it's not used anywhere else, you can leave the old value

Reply:
Yes, this is the protobufs PR: https://github.com/meshtastic/protobufs/pull/598

Reply:
I think this is why `M5STACK_COREBASIC` has been defined is protobufs. So we can use `M5STACK_COREBASIC` instead for `M5STACK` which is a brand and not a model to avoid confusion.

Reply:
Let's leave the hardware enum as it is, it's not showing up anywhere in writing. more like a macro for code...